models/player-model.js

```diff
@@ -13,13 +47,33 @@ var PlayerSchema = new Schema({
         'default': "",
         'required': true
     },
+    'city': {
```

Comment:
Please change the name of this variable to something more significant like city_of_birth.